src/styles/index.scss

```diff
@@ -1,3 +1,68 @@
+@import './vars';
+@import './extends';
+@import './mixins';
+@import './keyframes';
+
 body {
   margin: 0;
+  height: 100vh;
+
+  @extend %stopwatch-flex-property;
+}
+
+.stopwatch {
+  width: $stopwatch-size;
+  height: $stopwatch-size;
+  color: #000;
+
+  @extend %stopwatch-flex-property;
+  @include stopwatch-border-style(1vmin, dotted, 50%);
+
+  &__min {
+    @include stopwatch-elements-styles(
+      $stopwatch-min-width,
+      $stopwatch-min-height,
+      $stopwatch-colors-min
+    );
+
+    animation: hand-rotate 3600s infinite steps(60);
+    bottom: 50%;
+    transform-origin: bottom;
+    position: absolute;
+
+    @extend %stopwatch-flex-property;
+  }
+
+  &__second {
+    @include stopwatch-elements-styles(
+      $stopwatch-second-width,
+      $stopwatch-second-height,
+      $stopwatch-colors-second
+    );
+
+    animation: hand-rotate 60s infinite linear;
```

Comment:
The `hand-rotate` animation is used here, but it's not clear what it does from the name alone. Consider adding a comment above where the animation is defined to explain what it does.